templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -80,7 +80,7 @@ contents:
     # Path to the seccomp.json profile which is used as the default seccomp profile
     # for the runtime. If not specified, then the internal default seccomp profile
     # will be used.
-    seccomp_profile = ""
+    seccomp_profile = "/etc/crio/seccomp.json"
```

Comment:
Oh, wait.  This isn't a revert of 606bd2bc8a (#1414).  Before #1414, this line was `# seccomp_profile...` (commented out).  In #1414 it became master's current empty string.  With this commit it is becoming an uncommented, nonempty value.  So was there a default baked into CRI-O at some point, but no longer?  And now we need to provide the old default explicitly here to support CRI-O that are recent enough to have dropped the baked-in-default?  But bug 1829642 is about supporting older CRI-O?  I'm confused about how these pieces fit together.

Reply:
it was commented out further back: https://github.com/openshift/machine-config-operator/commit/69025e8e8c82ed6d188eb0e409e8148da09ac3b2#diff-5c79cb2bed55d5971e8178e82d18cdc9

Reply:
Ah, ok.  And looks like #1216 went out in 4.3 and was never backported to 4.2.

Reply:
pretty sure that 4.2/1/.12 fails if these are empty so it cant be backported

Reply:
Yeah this is needed, so that cri-o 1.14 can use this config and not fail to start up like it was.